### ISSUE TITLE
NullReferenceException: Error -  EditorZoomArea.cs:137

### ISSUE THREAD

**Issue by K-Mag** [mapbox/mapbox-unity-sdk#1074]
_Note: If this is a bug or support ticket, please provide the following information:_
* Unity version: Unity 2018.2.10f1 Personal
  * Scripting Runtime Version: .Net 3.5 Equivalent
  * Scripting Backend: Mono
  * Api Compatibility Level: .Net 2.0 Subset
* Mapbox SDK version: v1.4.5
* The platform you're building to: PC, Mac & Linux 
* A description of what you're trying to do: I only want to start the Example-Projects, but get this Error. 
* Steps to recreate the bug if appropriate: I didn't change anything. 
* Links to your logs are also helpful: 

Complete Error: 

NullReferenceException: Object reference not set to an instance of an object
Mapbox.Editor.NodeEditor.GUIScaleUtility.BeginScale (UnityEngine.Rect& rect, Vector2 zoomPivot, Single zoom, Boolean adjustGUILayout) (at Assets/Mapbox/Unity/Editor/NodeEditor/EditorZoomArea.cs:137)
Mapbox.Editor.NodeEditor.NodeBasedEditor.OnGUI () (at Assets/Mapbox/Unity/Editor/NodeEditor/NodeBasedEditor.cs:160)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:232)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MethodBase.cs:115)
UnityEditor.HostView.Invoke (System.String methodName, System.Object obj) (at /Users/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:295)
UnityEditor.HostView.Invoke (System.String methodName) (at /Users/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:288)
UnityEditor.HostView.InvokeOnGUI (Rect onGUIPosition) (at /Users/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:255)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

**Comment by K-Mag** [comment]
Solved by change **Scripting Runtime Version: .Net 4.x**

**Comment by K-Mag** [comment]
Found out, that I only get this error when Map Editor is open.

**Comment by jordy-isaac** [comment]
@K-Mag Can you provide some more details on what you're doing when you get this? We'll look into it.

**Comment by K-Mag** [comment]
Hey @jordy-isaac, 
Thanks for your reply. 
I found out, that I get the Error when I start the Map Editor. 
The Editor is empty and I get 999+ Errors when moving the window. 

I tried it on two different systems (Unity 2018.10f1 and Unity 2018.11f1) both MacOS. 
I even get the Error on each sample Project. 

I'm just open a new empty project. 
Import Mapbox SDK 1.4.5. 
Paste my Access Token and choose a sample project. 
When I start the Map Editor, the Error cames ... 

<img width="1318" alt="bildschirmfoto 2018-10-11 um 11 58 15" src="https://user-images.githubusercontent.com/30326885/46796766-b9e88780-cd4d-11e8-9294-054cd76f675d.png">

Thanks for helping
